src/pages/console/uibuilder/collections.mdx

```diff
@@ -1,19 +1,19 @@
 export const meta = {
   title: `Working with collections`,
-  description: `Collections`,
+  description: `Collections`
 };
 
 Collections are any list of repeating items. You can visually create a collection from any individual component by selecting the **Create collection** button in the top right corner of the UI component editor. All collections are generated as code with real-time subscriptions automatically set up.
 
 ## Data binding
 
-Amplify Studio offers a visual way to bind collections to items in your data model. Collection items can be filtered, sorted, or linked to specific records from the content management view.
+[Amplify Studio](/console/adminui/start/#logging-in-and-creating-an-app) offers a visual way to bind collections to items in your data model. Collection items can be filtered, sorted, or linked to specific records from the content management view.
```

Comment:
Rather than linking to the login at the first mention of Studio, I recommend moving the link to the first instances of working inside studio. Like right above "create a collection", "In Amplify Studio..."